src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -59,5 +57,8 @@ public Subsystems(Hardware h) {
         if (SHOOTER_ENABLED)
             shooterSubsystem = new ShooterSubsystem(hardware.flywheelMotor1, hardware.flywheelMotor2,
                     hardware.turretMotor, hardware.hoodMotor);
+        if (SHOOTER_VISION_ENABLED)
+            shooterVisionSubsystem = new ShooterVisionSubsystem(hardware.navX,
```

Comment:
Would it make sense to change `Hardware` so that the gyroscope would be connected if either drivebase or shooter vision are? I think the main question is if we would connect the gyroscope and not the rest of drivebase.

Reply:
I'm currently changing `Hardware` to initialize the gyroscope if shooter vision is enabled, I can change it if you want.